great_expectations/expectations/expectation.py

```diff
@@ -2349,35 +2350,38 @@ def _validate_metric_value_between(  # noqa: C901 - 21
         return {"success": success, "result": {"observed_value": metric_value}}
 
 
+@public_api
 class QueryExpectation(TableExpectation, ABC):
     """Base class for QueryExpectations.
 
-     QueryExpectations *must* have the following attributes set:
-         1. `domain_keys`: a tuple of the *keys* used to determine the domain of the
-            expectation
-         2. `success_keys`: a tuple of the *keys* used to determine the success of
-            the expectation.
+    QueryExpectations facilitate the execution of SQL or Spark-SQL queries as the core logic for an Expectation.
 
-    QueryExpectations *may* specify a `query` attribute, and specify that query in `default_kwarg_values`.
-    Doing so precludes the need to pass a query into the Expectation, but will override the default query if a query
-    is passed in.
+    QueryExpectations must implement a `_validate(...)` method containing logic for determining whether data returned by the executed query is successfully validated.
 
-     They *may* optionally override `runtime_keys` and `default_kwarg_values`;
-         1. runtime_keys lists the keys that can be used to control output but will
-            not affect the actual success value of the expectation (such as result_format).
-         2. default_kwarg_values is a dictionary that will be used to fill unspecified
-            kwargs from the Expectation Configuration.
+    Query Expectations may optionally provide implementations of:
+        1. `validate_configuration`, which should raise an error if the configuration
+            will not be usable for the Expectation
+        2. Data Docs rendering methods decorated with the @renderer decorator.
 
```

Comment:
Just need some whitespace to get the rendering to work here:
```suggestion
    Query Expectations may optionally provide implementations of:

    1. `validate_configuration`, which should raise an error if the configuration will not be usable for the Expectation.

    2. Data Docs rendering methods decorated with the @renderer decorator.

```